pages/index.vue

```diff
@@ -81,19 +86,34 @@ const playItem = (item: TrackModel, group: IDiscoverableGroup) => {
     <template v-else>
       <template v-for="group in discoverGroups" :key="group.header?.id || 0">
         <PageHeading v-if="group.header" :level="3">
-          <a
-            v-if="typeof group.header.link === 'string'"
-            :href="group.header.link"
-          >
-            {{ group.header.title }}
-          </a>
-          <span v-else>{{ group.header.title }}</span>
+          <div class="flex items-center justify-between">
+            <div>
+              <a
+                v-if="typeof group.header.link === 'string'"
+                :href="group.header.link"
+              >
+                {{ group.header.title }}
+              </a>
+              <span v-else>{{ group.header.title }}</span>
+            </div>
+            <NuxtLink v-if="group.header.link" :to="group.header.link">
+              <ButtonStyled intent="secondary" size="small">
+                <span
+                  class="overflow-hidden text-ellipsis whitespace-nowrap text-center"
```

Comment:
Could we remove the `overflow-hidden` here? Is it actually needed after all when having `whitespace-nowrap` and `text-ellipsis` in place, doesn't this contradict each other?

Actually, what about removing all of the mentioned? Shouldn't this button be to be read in its full text?

Reply:
whitespace-nowrap is required to keep the button height for small screen. But the other classes can be removed